azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -207,18 +207,22 @@ def get_file_map(
     file_map = {"__namespaces__": {}}
     if mq_traces and path.join(ops_path, "traces") in walk_result:
         # still possible for no traces if cluster is too new
-        assert len(walk_result) == 2
+        assert len(walk_result) == 3
         assert walk_result[ops_path]["folders"]
         assert not walk_result[path.join(ops_path, "traces")]["folders"]
         file_map["traces"] = convert_file_names(walk_result[path.join(ops_path, "traces")]["files"])
     elif ops_service == "billing":
-        assert len(walk_result) == 2
+        assert len(walk_result) == 3
         ops_path = path.join(BASE_ZIP_PATH, namespace, ops_service)
         c_path = path.join(BASE_ZIP_PATH, c_namespace, "clusterconfig", ops_service)
         file_map["usage"] = convert_file_names(walk_result[c_path]["files"])
         file_map["__namespaces__"]["usage"] = c_namespace
-    elif ops_service != "otel":
+    elif ops_service == "deviceregistry":
         assert len(walk_result) == 1
```

Comment:
so if assets + endpoints are not created, this should be one (meta folder) but if they are created, this should be 2?

Reply:
right, update and verify as we talked offline

Reply:
updated and verified the IT pass with/without DR crd